config/routes.rb

```diff
@@ -4,6 +4,8 @@
 
   resource :questions, only: [:create]
 
+  resource :answers, only: [:create]
```

Comment:
Se tu não for usar depois, tu pode apagar as outras actions do controller, no controller de questions também